packages/core/src/data-editor/data-editor.tsx

```diff
@@ -2447,8 +2446,11 @@ const DataEditorImpl: React.ForwardRefRenderFunction<DataEditorRef, DataEditorPr
         isActivelyDragging.current = false;
     }, []);
 
+    const hoveredRef = React.useRef<GridMouseEventArgs>();
     const onItemHoveredImpl = React.useCallback(
         (args: GridMouseEventArgs) => {
+            if (mouseEventArgsAreEqual(args, hoveredRef.current)) return;
```

Comment:
This is here to prevent spurious events being sent out to the user. Confirm?

Reply:
Yep; I put a console log statement there to check. It’s caused by the onItemHoveredImpl being called by onScroll.

Reply:
I basically copied the same logic from data-grid since it also has hoveredRef and onItemHovered.